modules/graphql-query-builder.js

```diff
@@ -123,3 +119,10 @@ function buildQuery(queryIdPrefix, queryIdSuffix) {
         throw "unrecognized query " + queryIdPrefix;
     }
 }
+
+function substitutePolicyCodeField(text, policyCodeFormat) {
+    if (!policyCodeFormat || policyCodeFormat === "xml") return text;
```

Comment:
is it == ?

Reply:
=== is more specific way of comparison. == leads to unnecessary type conversions.